example/android/app/src/main/java/com/ddsdkreactnativeexample/MainActivity.java

```diff
@@ -3,12 +3,11 @@
  * This product includes software developed at Datadog (https://www.datadoghq.com/).
  * Copyright 2016-Present Datadog, Inc.
  */
-
-package com.example.ddsdkreactnative;
+ 
+ package com.ddsdkreactnativeexample;
```

Comment:
minor formatting suggestion

```suggestion

package com.ddsdkreactnativeexample;
```